# 99-unnamed.dpatch by  <agomes@>
# All lines beginning with `## DP:' are a description of the patch.
# DP: See patch name

Index: mozilla/widget/src/gtk2/nsDragService.cpp
===================================================================
--- mozilla.orig/widget/src/gtk2/nsDragService.cpp
+++ mozilla/widget/src/gtk2/nsDragService.cpp
@@ -152,16 +152,19 @@
 nsDragService::InvokeDragSession(nsIDOMNode *aDOMNode,
                                  nsISupportsArray * aArrayTransferables,
                                  nsIScriptableRegion * aRegion,
                                  PRUint32 aActionType)
 {
     PR_LOG(sDragLm, PR_LOG_DEBUG, ("nsDragService::InvokeDragSession"));
     nsBaseDragService::InvokeDragSession(aDOMNode, aArrayTransferables,
                                          aRegion, aActionType);
+#ifdef MOZ_MICROB
+    return NS_ERROR_INVALID_ARG;
+#endif
     // make sure that we have an array of transferables to use
     if (!aArrayTransferables)
         return NS_ERROR_INVALID_ARG;
     // set our reference to the transferables.  this will also addref
     // the transferables since we're going to hang onto this beyond the
     // length of this call
     mSourceDataItems = aArrayTransferables;
     // get the list of items we offer for drags
